### ISSUE TITLE
UTF-8 support

### ISSUE THREAD

**Issue by cidadao** [thisbejim/Pyrebase#53]
Currently json.dumps() is used without additional arguments other than the data itself. Passing "ensure_ascii=False" ensures data is decoded with UTF-8, so no weird characters anymore.

**Comment by cidadao** [comment]
PS: let me know what you think of this :) I can make a PR with the required changes if you prefer since I'll need to send UTF-8 data to my Firebase project

**Comment by cidadao** [comment]
I'm currently stuck here:
```
Traceback (most recent call last):
  File "test_db.py", line 128, in <module>
    main()
  File "test_db.py", line 121, in main
    mode_file(args.file)
  File "test_db.py", line 109, in mode_file
    write_entry_to_db(group)
  File "test_db.py", line 54, in write_entry_to_db
    ret = firebase_db.child("groups").child(new_key).set(new_data)
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyrebase-3.0.7-py3.5.egg\pyrebase\pyrebase.py", line 237, in set
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\site-packages\requests\sessions.py", line 528, in put
    return self.request('PUT', url, data=data, **kwargs)
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\site-packages\requests\sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\site-packages\requests\sessions.py", line 585, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\site-packages\requests\adapters.py", line 403, in send
    timeout=timeout
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\site-packages\requests\packages\urllib3\connectionpool.py", line 578, in urlopen
    chunked=chunked)
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\site-packages\requests\packages\urllib3\connectionpool.py", line 362, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\http\client.py", line 1106, in request
    self._send_request(method, url, body, headers)
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\http\client.py", line 1150, in _send_request
    body = _encode(body, 'body')
  File "C:\Users\mribeiro\AppData\Local\Programs\Python\Python35-32\lib\http\client.py", line 161, in _encode
    (name.title(), data[err.start:err.end], name)) from None
UnicodeEncodeError: 'latin-1' codec can't encode character '\u2030' in position 192: Body ('▒') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.
```

The underlying HTTP client library "insists" on sending data with Latin-1 encoding instead of UTF-8.

**Comment by cidadao** [comment]
Solved! It turned out that you only need to add `encode("utf-8")` to `json.dumps()` output. Pull request done, let me know what you think.

**Comment by thisbejim** [comment]
Nice job figuring this all out @cidadao, this is now solved with your PR #54
